test/test_unary_ufuncs.py

```diff
@@ -495,6 +495,7 @@ def _test_out_arg(self, op, input, output, expected, **kwargs):
             else:
                 self.assertEqual(output, expected.to(output.dtype))
 
+    @unittest.skipIf(TEST_WITH_ROCM, "Skipping as we won't be fixing test_out_arg_all_dtypes_reciprocal_cuda_complex64 and test_out_arg_all_dtypes_sinc_cuda_complex64 test failure on an older version of pytorch")
```

Comment:
@rraminen @pruthvistony This line is both wrong and ineffective, it seems:
* wrong because the intention seems to have been to skip only specific variants of the test_out_arg_all_dtypes test, but the way it is done here would end up skipping **all** variants ie. "overskipping".
* ineffective because, luckily, the skipIf doesn't seem to have *any* effect and the tests still fail: https://ontrack-internal.amd.com/browse/SWDEV-447418?focusedId=15110847&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-15110847

Fix to skip the tests properly is in https://github.com/ROCm/pytorch/commit/eb5c7f905ae0207a785cbdd8c70c42c8d67e2c41